### PR TITLE
`azurerm_dashboard_grafana` - Support `grafana_integrations`

### PR DESCRIPTION
**Pull request by ms-zhenhua**
image

### CODE REVIEW

**Review by katbyte — CHANGES_REQUESTED** [review]

File: internal/services/dashboard/dashboard_grafana_resource.go [hunk 1]
```diff
@@ -86,6 +95,30 @@ func (r DashboardGrafanaResource) Arguments() map[string]*pluginsdk.Schema {
...
+									Type:         pluginsdk.TypeString,
+									Required:     true,
+									ValidateFunc: validation.StringIsNotEmpty,
+								},
+							},
+						},
+					},
+				},
+			},
+		},
```
[the remark this answers is not in the document]
Comment (ms-zhenhua) [reply internal/services/dashboard/dashboard_grafana_resource.go:120]: Hi @katbyte, thank you for your review.
To simplify the property names, I suggest renaming `azure_monitor_workspace_integrations` to `azure_monitor_workspace` and `azure_monitor_workspace_resource_id` to `resource_id`.
However, I suggest keeping the middle block of `azure_monitor_workspace`. The reason is **Azure Service Team** may add new properties under `azure_monitor_workspace` in future swagger.  If we keep `azure_monitor_workspace`, we only need to add new properties under `azure_monitor_workspace`, or we have to define a new `grafana_integrations` with a new `list` definition to replace the existing one, which is more complicated. WDYT?

[the remark this answers is not in the document]
Comment (katbyte) [reply internal/services/dashboard/dashboard_grafana_resource.go:120]: that works

**Comment by ms-zhenhua** [comment]
Hi @katbyte, I have resolved the comments. Kindly help review it again. Thanks.

**Review by katbyte — APPROVED** [review]
LGTM 🍄

**Pull request merged**